_bazaar/rse-in-the-digital-humanties-domain.md

```diff
@@ -1,13 +1,28 @@
 ---
-author:
-  links:
-  - icon: fas fa-fw fa-envelope-square
-    label: Email
-    url: mailto:rse@fthiery.de
-  name: Florian Thiery / Research Squirrel Engineers
+authors:
+  - links:
+    - icon: fas fa-fw fa-envelope-square
+      label: Email
+      url: mailto:hi@squirrel.link
+    - icon: fas fa-fw fa-link
+      label: Website
+      url: http://squirrel.link
+    - icon: fas fa-fw fa-twitter-square
+      label: Twitter
+      url: https://twitter.com/RSESquirrel
+    - icon: fas fa-fw fa-github-square
+      label: Github
+      url: https://github.com/research-Squirrel-Engineers
```

Comment:
```suggestion
  email: hi@squirrel.link
  uri: http://squirrel.link
  twitter: RSESquirrel
  github: research-Squirrel-Engineers
```
it's not documented, but these are all implemented in the [author-profile](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/author-profile.html)